test/common/runtime/runtime_impl_test.cc

```diff
@@ -881,6 +881,17 @@ class RtdsLoaderImplTest : public LoaderImplTest {
     VERBOSE_EXPECT_NO_THROW(rtds_callbacks_[0]->onConfigUpdate(decoded_resources.refvec_, {}, ""));
   }
 
+  void doDeltaOnConfigRemovalVerifyNoThrow(const std::string& resource_name) {
+    Protobuf::RepeatedPtrField<std::string> removed_resources;
+    *removed_resources.Add() = resource_name;
+    VERBOSE_EXPECT_NO_THROW(rtds_callbacks_[0]->onConfigUpdate({}, removed_resources, ""));
+  }
+
+  void doDeltaOnConfigRemoval(const std::string& resource_name) {
+    Protobuf::RepeatedPtrField<std::string> removed_resources;
+    *removed_resources.Add() = resource_name;
+  }
```

Comment:
This seems to not be doing anything: all we're doing is creating a temporary proto and throw it away. Ddi you mean to call `onConfigUpdate` here?

Reply:
i was rushing and pushed a local version 😬, the right one coming up